### ISSUE TITLE
Autocomplete mentions without requiring @

### ISSUE THREAD

**Issue by raine** [gitterHQ/gitter#615]
A lot of oldschool IRC users have the muscle memory of autocompleting nicks without at sign.

I can't see any reason why `gitter<TAB> hello` couldn't expand to `@gitter hello`.

Having to type @ is an inconvenience in itself, so this would benefit everyone.

**Comment by trevorah** [comment]
Btw, if you still have a hankering for the oldschool: https://irc.gitter.im. @malditogeek is pushing out the first [open source](https://github.com/gitterHQ/irc-bridge) deployment of it right now.

**Comment by malditogeek** [comment]
Yeah, https://github.com/gitterHQ/irc-bridge is now in production :tada: 

Gonna close this issue as is a duplicate of gitterhq/gitter#158, will probably do something about it soon.

**Comment by raine** [comment]
That is not a duplicate of this issue.

This is for gitter web UI, #158 for IRC clients.

Sent from my iPhone

--
Raine

**Comment by malditogeek** [comment]
Oops, I thought you meant to modify this behaviour on IRC, reopening the issue

**Comment by oKcerG** [comment]
Related to this issue, but another suggestion to deal with it:
Why is the @ needed in the first place ? Couldn't we just do highlights when a plain username is mentioned, like good old IRC ?

**Comment by mydigitalself** [comment]
A few points to consider: 

1. This is inherited behaviour from GitHub, where we have inherited a lot of behaviour
2. In IRC it's a client-side behaviour, right? Our servers process this information and deal with it accordingly, which means if somebody said something over IRC, then we have to process each and every word for an implicit mention. This becomes even more of a problem when you consider the next point...
3. This person: https://github.com/the and this person: https://github.com/and and this person https://github.com/but etc... how do we distinguish between just words and explicit mentions, and the only way I can see that happening is through making it explicit.

**Comment by mydigitalself** [comment]
Didn't mean to close the issue, happy to listen to more opinion on the matter.

**Comment by raine** [comment]
I don't mind the @ and there seem to be good reasons for having it, but I do think autocomplete should work without it.

**Comment by abhisekp** [comment]
I also think `gitter<tab>` should expand to `@gitter`. Why waste a `tab` keybinding ;)
